app/utils/getNews.ts

```diff
@@ -0,0 +1,16 @@
+import { schemaNewsResponse } from '@/models/schemes';
+import { NewsResponse } from '@/models/types';
+import axios from 'axios';
+
+const getNews = async (keyword: string): Promise<NewsResponse> => {
```

Comment:
Please remove :Promise<NewsResponse> because TS implicitly gets the type out of zod